src/cudamatrix/cu-matrix.cc

```diff
@@ -1229,6 +1229,47 @@ void CuMatrixBase<Real>::AddMatMatElements(Real alpha,
   }
 }
 
+template<typename Real>
+void CuMatrixBase<Real>::ParametricReLU(const CuMatrixBase<Real> &src, const CuVectorBase<Real> &a, const CuVectorBase<Real> &b) {
+  KALDI_ASSERT(SameDim(*this, src));
```

Comment:
Please add asserts:
KALDI_ASSERT(a.Dim() == *this->NumCols());
KALDI_ASSERT(b.Dim() == *this->NumCols());